### ISSUE TITLE
[Bug Report] Expansion panel does not open when wrapped in custom component

### ISSUE THREAD

**Issue by sindrepm** [vuetifyjs/vuetify#2617]
### Versions and Environment
**Vuetify:** 0.17.2
**Vue:** 2.5.7
**Browsers:** Chrome 62.0.3202.94
**OS:** Windows 7

### Steps to reproduce
I am unsure whether this should be filed as a bug or feature request, but it appears to me as a bug.

Basically I would like to wrap my expansion panels in custom components in order to add some extra functionality and keep it clean and separate. However, when doing this the panel won't open when I click on the header.

See provided codepen.

### Expected Behavior
The panel should open

### Actual Behavior
The panel does not open

### Reproduction Link
[https://codepen.io/anon/pen/RjydZy](https://codepen.io/anon/pen/RjydZy)

### Other comments
The resulting markup is the same as when just using the Vuetify components directly, i.e. my component is not adding any additional nesting.

As a workaround I have added by own @click handler to the header slot element which simply toggles the v-model. This kind of works, but has some issues, mainly:

- Not firing when clicking on the up/down arrow
- Not collapsing on "click outside"
- Header padding cause the clickable area to be less than the visible header area

Is this something that can be fixed or is it a limitation of Vue?

**Comment by csicky** [comment]
Any news on this? Just found out it doesn't open on click after I spent a few hours in making the custom component.

**Comment by csicky** [comment]
I managed to hack it :) I created a custom `getChildren` for `this.$parent` to count my custom elements, created a custom `toggle` method, event handler for header, my uids are strings to filter the noise, `onClick() { this.$parent.panelClick(this.uid) } `, the header had to be re-styled to expand and the action icon had to be re-added as absolute positioned and styled to be under the header to not hijack the click.

Best part is now I see that it is assigned now :) I'll revert my hacks when it's fixed. Thank you for working on it.
